dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/components/localfiletreetable/FileTreeTableItem.java

```diff
@@ -119,7 +119,7 @@ public FontAwesomeIconView getGraphicFont(final FileTreeModel fileTreeModel) {
                 super.setGraphic(previousGraphics);
             }));
             buildChildren.setOnFailed(SafeHandler.logHandle(event -> {
-                LOG.info("Success");
+                LOG.info("Failed to build children");
```

Comment:
This should be an error log.  Also we're not logging the exception.  And we probably should describe more about what it means by 'children.'